File: 1859-sorting-the-sentence/1859-sorting-the-sentence.js
/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function(s) {
    let words = s.split(" ");
    let result = new Array(words.length).fill(0);
    for(let word of words){
        let index = word.substring(word.length-1) - '0';
        result[index-1] = word.substring(0,word.length-1);
    }
    return result.join(" ");
};

File: 1859-sorting-the-sentence/1859-sorting-the-sentence.java
class Solution {
    public String sortSentence(String s) {
        String[] words = s.split(" ");
        String[] result = new String[words.length];
        for(String str: words){
            int index = str.charAt(str.length () -1) - '0';
            result[index-1] = str.substring(0, str.length()-1);
        }
        return String.join(" ", result);
    }
} 